packages/dds/sequence/src/sequence.ts

```diff
@@ -230,9 +230,9 @@ export abstract class SharedSegmentSequence<T extends ISegment>
      * @param start - The inclusive start of the range to remove
      * @param end - The exclusive end of the range to remove
      */
-    public removeRange(start: number, end: number) {
+    public removeRange(start: number, end: number, sendOp: boolean = true) {
```

Comment:
this doesn't really work, as merge tree will expect and op to be generated and sent. merge tree will have a segment group that tracking the ack of that operation. i think you need to push all the rollback code down to client/merge-tree where you have more control over the internal structure. 

you need to call merge tree's markRangeRemoved with the following params as constants
refSeq: UniversalSequenceNumber
clientId: LocalClientId
seq: TreeMaintenanceSequenceNumber